### ISSUE TITLE
Sub-packages of pandas do not appear

### ISSUE THREAD

**Issue by drorata** [microsoft/python-language-server#1870]
I don't know whether it's a bug or not, but please consider the following behavior:

![pandas_sub_package](https://user-images.githubusercontent.com/2905978/72257682-85c4dc00-360c-11ea-97f2-830a8fd964a0.gif)

First, I import `pandas` and it all looks fine. Next, I want to type `from pandas.io.json import json_normalize` and it works nicely as well. Lastly, I want to try `pd.io.json.json_normalize` directly, and this is where the behavior is unfriendly. The LS doesn't find neither `io` nor `json` when I start typing `pd.`.

So, is this a bug or an expected behavior? If it's neither, how can I workaround it?

**Comment by MikhailArkhipov** [comment]
This is b/c `pandas` has `io` submodule and hence `pandas.io` works in import statement completion. However, `__init__.py` does not directly import `io` and only has `from pandas.io.api ...`.  So `io` does not appear in exported items. I guess submodules should be added there too.

**Comment by drorata** [comment]
What `__init__.py` are you referring to?

**Comment by MikhailArkhipov** [comment]
The one in `site-packages/pandas` folder.

**Comment by drorata** [comment]
So is this a `pandas` bug??

**Comment by MikhailArkhipov** [comment]
No, LS completion issue. I think submodules should appear in the list even if not explicitly imported.

**Comment by drorata** [comment]
@MikhailArkhipov Can it be that a similar problem happens with `numpy.testing`?

**Comment by MikhailArkhipov** [comment]
Unfortunately, change regressed more than it fixed. Reverting, needs more work.
